models/user.js

```diff
@@ -14,14 +16,33 @@ userSchema = mongoose.Schema({
     email: {
         type: String,
         lowercase: true,
-  //      unique: true,
+    },
+    password: {
+        type: String
+    },
+    role: {
+        type: String,
+        enum: ["candidate", "recruiter"]
+    },
+    candidateProfile: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "Candidate"
+    }, 
+    jobOffers: [{
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "Job"
+    }],
     }
```

Comment:
When I tried to run this app, I got an error message for this line. This "}" is probably wrong

Reply:
oh thank you! forgot to pull this typo